docker/build_push.sh

```diff
@@ -85,9 +85,8 @@ fi
 
 #if a prebuild, always -1, else if "docker pull" failed build the image.
 if [ "$PULLED" != "0" ]; then
+  export LIBRA_BUILD_TAG=${PRE_NAME}
```

Comment:
should we unset this after running docker/$DIR/build.sh?

Reply:
Yes for added precaution. I think we are fine without it though. Most of us won't use this script but CI.  In CI, https://github.com/libra/libra/blob/master/.circleci/config.yml, the next call to build_push.sh will override this env var.